python/aswfdocker/tests/test_builder.py

```diff
@@ -0,0 +1,121 @@
+import unittest
+import logging
+import tempfile
+
+from click.testing import CliRunner
+
+from aswfdocker import builder, buildinfo, constants
+from aswfdocker.cli import aswfdocker
+
+
+class TestBuilder(unittest.TestCase):
+    def setUp(self):
+        self.maxDiff = None
+        self.build_info = buildinfo.BuildInfo(
+            repo_uri="notauri", source_branch="testing", aswf_version="2019.123"
+        )
+
+    def test_package_baseqt_2019_dict(self):
+        b = builder.Builder(
+            self.build_info,
+            builder.GroupInfo(
+                name="baseqt", version="2019", type_=constants.ImageType.PACKAGE,
+            ),
+        )
+        self.assertEqual(
+            b.make_bake_dict(),
+            {
+                "group": {"default": {"targets": ["package-qt"]}},
+                "target": {
+                    "package-qt": {
+                        "context": ".",
+                        "dockerfile": "packages/Dockerfile",
+                        "args": {
+                            "ASWF_ORG": "aswflocaltesting",
+                            "ASWF_PKG_ORG": "aswftesting",
+                            "ASWF_VERSION": constants.VERSIONS[
+                                constants.ImageType.PACKAGE
+                            ]["qt"][1],
+                            "BUILD_DATE": "dev",
```

Comment:
These arguments seem to be duplicated from other locations, for instance packages/Dockerfile (I think? See [issue 27](https://github.com/AcademySoftwareFoundation/aswf-docker/issues/27). Should they be pulled from a configuration file?